<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBase</class>
 <widget class="QWidget" name="frmBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widTop" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>27</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widTopLeft" native="true">
            <property name="minimumSize">
             <size>
              <width>8</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>8</width>
              <height>27</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widTitle" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labIcon">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>22</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="lineWidth">
                  <number>0</number>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                 <property name="indent">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labTitle">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widButton" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>27</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>27</height>
                  </size>
                 </property>
                 <widget class="QPushButton" name="btnMin">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>0</y>
                    <width>28</width>
                    <height>19</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>28</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>28</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../EfbManagement.qrc">
                    <normaloff>:/img/MinSizeButton.png</normaloff>:/img/MinSizeButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>28</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnMax">
                  <property name="geometry">
                   <rect>
                    <x>38</x>
                    <y>0</y>
                    <width>26</width>
                    <height>19</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>26</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>26</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../EfbManagement.qrc">
                    <normaloff>:/img/MaxSizeButton.png</normaloff>:/img/MaxSizeButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>26</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnClose">
                  <property name="geometry">
                   <rect>
                    <x>64</x>
                    <y>0</y>
                    <width>27</width>
                    <height>19</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>27</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>27</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../EfbManagement.qrc">
                    <normaloff>:/img/CloseButton.png</normaloff>:/img/CloseButton.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>27</width>
                    <height>19</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widTopRight" native="true">
            <property name="minimumSize">
             <size>
              <width>9</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>9</width>
              <height>27</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widMid" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widLeft" native="true">
            <property name="minimumSize">
             <size>
              <width>6</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>6</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widCenter" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widContent" native="true">
                 <property name="font">
                  <font>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widStatus" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>23</height>
                  </size>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="Line" name="line">
                      <property name="orientation">
                       <enum>Qt::Horizontal</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>2</number>
                      </property>
                      <item>
                       <widget class="QLabel" name="labBarLeft">
                        <property name="maximumSize">
                         <size>
                          <width>16666666</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>9</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="labBarRight">
                        <property name="minimumSize">
                         <size>
                          <width>200</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>200</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>9</pointsize>
                         </font>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widRight" native="true">
            <property name="minimumSize">
             <size>
              <width>6</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>6</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widBut" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>9</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>9</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widButtomLeft" native="true">
            <property name="minimumSize">
             <size>
              <width>8</width>
              <height>9</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>8</width>
              <height>9</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widButtom" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>9</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>9</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widButtomRight" native="true">
            <property name="minimumSize">
             <size>
              <width>9</width>
              <height>9</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>9</width>
              <height>9</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../EfbManagement.qrc"/>
 </resources>
 <connections/>
</ui>
